x-pack/legacy/plugins/reporting/server/lib/level_logger.ts

```diff
@@ -46,7 +46,11 @@ export class LevelLogger implements Logger {
     this._logger([...this._tags, ...tags, 'info'], msg);
   }
 
+  public get isVerbose() {
+    return this._isVerbose;
+  }
+
   public clone(tags: string[]) {
-    return new LevelLogger(this._logger, [...this._tags, ...tags]);
+    return new LevelLogger(this._logger, [...this._tags, ...tags], this._isVerbose);
```

Comment:
In our code, `server/browsers/chromium/driver_factory/index.ts` references `logger.isVerbose`. As far as I could tell, that field was never initialized. Important to note that it still isn't (but now Typescript is happy).